deploy.sh

```diff
@@ -0,0 +1 @@
+chmod -Rf 777 data/DoctrineORMModule/Proxy
```

Comment:
Having a `deploy.sh` (or `doctrine-init.sh`) file to automate setup/deployment is good. But it makes sense to include everything that's needed to setup the system, so adding the following line would be good too:
```
vendor/bin/doctrine-module orm:schema-tool:update --force
```